python/llm/example/transformers/int4_pipeline.py

```diff
@@ -62,27 +45,27 @@ def inference(llm, repo_id_or_model_path, model_family, prompt):
         # Option 1: Use HuggingFace transformers tokenizer
         print('-'*20, ' HuggingFace transformers tokenizer ', '-'*20)
         
-        print('Please note that the loading of HuggingFace transformers tokenizer may take some time.\n')
+        # print('Please note that the loading of HuggingFace transformers tokenizer may take some time.\n')
         # here is only a workaround for default example model 'decapoda-research/llama-7b-hf' in LLaMA family,
         # due to its out-of-date 'tokenizer_class' defined in its tokenizer_config.json.
         #
         # for most cases, you could use `AutoTokenizer`.
-        if model_family == 'llama':
-            from transformers import LlamaTokenizer
-            tokenizer = LlamaTokenizer.from_pretrained(repo_id_or_model_path)
-        else:
-            from transformers import AutoTokenizer
-            tokenizer = AutoTokenizer.from_pretrained(repo_id_or_model_path)
+        # if model_family == 'llama':
```

Comment:
1. Shall we still demonstrate the way using huggingface tokenizer? User needs to specify two different paths for tokenizer and model. 
The usage will be like
```python
           from bigdl.llm.transformers import BigdlForCausalLM
               llm = BigdlForCausalLM.from_pretrained("/path/to/ggml/bin",...)
            from transformers import LlamaTokenizer
            tokenizer = LlamaTokenizer.from_pretrained('/path/to/huggingface/repo')

            tokens_id = tokenizer(prompt).input_ids
            output_tokens_id = llm.generate(tokens_id, max_new_tokens=32)
            output = tokenizer.batch_decode(output_tokens_id)            
```

2. Shall we separate conversion and load/inference in the example, or just remove the convert and write it in README? Support the user will convert once and load/inference many times?

3. Do we still support download and loading pth/gptq models directly from huggingface using `BigdlForCausalLM`? Will it be confusing? Currently I removed them so user has to explicitly convert the model and then use `BigdlForCausalLM` to load it. 

@jason-dai